medcat/config_meta_cat.py

```diff
@@ -28,7 +27,7 @@ class General(MixingConfig, BaseModel):
     """Number of annotations to be meta-annotated at once in eval"""
     annotate_overlapping: bool = False
     """If set meta_anns will be calcualted for doc._.ents, otherwise for doc.ents"""
-    tokenizer_name: str = 'bbpe'
+    tokenizer_name: str = 'bert-tokenizer'
```

Comment:
Changing the default here could change existing behaviour. We want to avoid that.

Reply:
I've reverted the change, I made bert as the default one because I wanted to use it for meta_cat_tests.